lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -85,6 +85,7 @@ def build_purchase_authorize_request(post, money, payment, options)
         add_payment(post, payment, options)
         add_purchase_data(post, money, payment, options)
         add_fraud_detection(post, options)
+        add_wallet_id(post, options)
```

Comment:
I notice that most of the fields that `wallet_id` gets grouped with in the original `decidir` gateway are located in `add_purchase_data`. Any particular reason to add it via this separate method instead of including in the existing method? I don't necessarily have a preference either way, just curious

Reply:
hmmm, there wasnt much reasoning behind it! I just saw that we add fraud detection separately and thought itd be fine to do so for the wallet_id but I can place it inside the add_purchase_data instead!